### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gibbscullen**
**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,18 +15,10 @@ More information:
 
 ## Community meetings
 
-M3 contributors and maintainers have monthly (every four weeks) meetings at 11:00 AM (USA Pacific) on Tuesday.
+We are currently reworking the structure and schedule for M3 Community Meetings. The next meeting is currenty schedule for Tuesday, May 19th. Stay tuned for more information!
```
Comment (arnikola) [inline README.md:18]: Can we include a zoom link / calendar invite? (Also add the same to the headers in the #m3 zoom and gitter rooms)

Comment (gibbscullen) [reply README.md:18]: We will once we have more details.. looking into a better solution currently.

Comment (arnikola) [reply README.md:18]: Fair enough, can we link the #m3 slack here for now then? 

Could we do a provisional calendar invite that we can append details to later so people can subscribe to it for now?

Comment (gibbscullen) [reply README.md:18]: Yeah, I will add the Slack link. Need to hold off on calendar invite for now until we confirm method.

**Comment by gibbscullen** [comment]
No longer needed

**Pull request closed without merging**